appium/webdriver/webdriver.py

```diff
@@ -523,9 +523,23 @@ def update_settings(self, settings):
 
     @property
     def device_time(self):
-        """Returns the date and time from the device
+        """Returns the date and time from the device.
         """
-        return self.execute(Command.GET_DEVICE_TIME, {})['value']
+        return self.execute(Command.GET_DEVICE_TIME_GET, {})['value']
+
+    def get_device_time(self, format=''):
+        """Returns the date and time from the device. (Only available since Appium 1.11.0)
```

Comment:
TODO: Need to check next released Appium server version which includes https://github.com/appium/appium-base-driver/pull/294

Reply:
Thanks. Yes, I've checked behavior with my changes in my local.
My TODO means I'll need to update version on comment if next version is different from 1.11.0. (e.g. 1.10.1)

If my understanding is wrong, please let me know.

Thanks.

Reply:
Thanks, I've removed 'DO NOT MERGE' from issue's title.